### ISSUE TITLE
How much to lighten notebooks?

### ISSUE THREAD

**Issue by MatteoRobbiati** [qiboteam/tutorials#2]
At present, the strategy is to clean the output of notebooks by using an appropriate `.gitconfig`. 
I am opening this issue because although this is a working solution, it may be desirable to further lighten the files.

**Comment by AleCandido** [comment]
This is already being investigated in https://github.com/qibogang/qibogang.github.io/issues/28

**Comment by MatteoRobbiati** [comment]
> This is already being investigated in [qibogang/qibogang.github.io#28](https://github.com/qibogang/qibogang.github.io/issues/28)

Thank you for citing the other issue. I think it can be useful to keep this issue open too (in order to track the situation).

**Comment by AleCandido** [comment]
I agree: this will be the final repo where things are going to happen.

However, this is still at investigation stage, and related to the website. So I'd keep doing the investigation there for another while.
Later on, we will move everything related to tutorials here (but loading, that is always happening on the website side).
